src/__forks__/Promise.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Copyright 2013-2014 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Where is this file coming from that it is Apache licensed? RN and this are both BSD so we should make sure we have the appropriate header here.